qa/vagrant/src/test/resources/packaging/scripts/plugins.bash

```diff
@@ -107,3 +107,14 @@ install_and_check_plugin() {
         assert_file_exist "$(readlink -m $ESPLUGINS/$fullName/$file)"
     done
 }
+
+# Compare a list of plugin names to the plugins in the plugins pom and see if they are the same
+# $1 the file containing the list of plugins we want to compare to
+compare_plugins_list() {
+    cat $1 | sort > $1
+    local expected_file_name="$(echo $RANDOM)"
```

Comment:
I don't think you need a random file here - just a specific file is fine. Remember that we'll just nuke the file and start over the next time you execute this. We're fine with this kind of thing because these are mostly run in virtual machines. I think we are, anyway.